full_bubble_model.py

```diff
@@ -941,17 +982,17 @@ def plot(cpar, t_int=np.array([0.0, 1.0]), n=5.0, base_name='', format='png', LS
     if type(cpar) == dict:
         cpar = dotdict(cpar)
 
-    num_sol, error_code, elapsed_time = solve(cpar, t_int, LSODA_timeout, Radau_timeout, extra_dims)
+    num_sol, error_code, elapsed_time = solve(cpar, t_int, LSODA_timeout, Radau_timeout, extra_dims, print_errors=True)
     data = get_data(cpar, num_sol, error_code, elapsed_time)
     
 # Print errors
     errors, success = get_errors(error_code, printit=True)
-    if not success:
```

Comment:
from now on, plot() will draw a graph until the last succesful step if convergence failiure is encountered
error messages are printed during fatal errors